src/Matrix.cc

```diff
@@ -2200,7 +2200,7 @@ NAN_METHOD(Matrix::TemplateMatches) {
 
 // @author ytham
 // Match Template filter
-// Usage: output = input.matchTemplate("templateFileString", method);
+// Usage: output = input.matchTemplate("templateFileString", ["method"], ["maskPath"]);
```

Comment:
I guess you mean the `[]` notation to mean 'optional'? But it's confusing in javascript because it looks like you mean `Array`